packages/metatransaction-broadcaster/MetatransactionBroadcaster.js

```diff
@@ -234,6 +234,25 @@ class MetatransactionBroadcaster {
       // Not a voting rep related transaction (we recognise)
     }
 
+    const multicallDef = await this.loader.load({ contractName: "Multicall" }, { abi: true, address: false });
+    const possibleMulticall = new ethers.Contract(target, multicallDef.abi, this.wallet);
+
+    try {
+      const tx = possibleMulticall.interface.parseTransaction({ data: txData });
+      if (tx.signature === "multicall(bytes[])") {
+        // We check for each multicall whether it's doing something we'd allow
```

Comment:
Setting `const actions = tx.args[0]` would make the `for` loop a bit more readable, and you could use `actions[i]` pretty clearly.